demos/projects/ESPRESSIF/esp32/config/demo_config.h

```diff
@@ -109,26 +109,36 @@
  * @brief Device symmetric key
  *
  */
+#ifdef CONFIG_AZURE_IOT_DEVICE_SYMMETRIC_KEY
```

Comment:
As a follow-up, these changes to the other ESP samples should probably be made in the ADU project as well